hyp3_sdk/asf_search.py

```diff
@@ -1,80 +1,102 @@
+
 _SEARCH_API = 'https://api.daac.asf.alaska.edu/services/search/param'
 
 
 def get_metadata(granules: Union[str, Iterable[str]]) -> Union[dict, List[dict]]:
     """Get the metadata for a granule or list of granules
 
     Args:
-        granules: granule(s) to lookup metadata for
+        granules: granule(s) to lookup
 
     Returns:
-        metadata: metadata for the granule(s)
+        metadata: GeoJSON Feature or FeatureCollection of the requested granule(s)
     """
+    warnings.warn('hyp3_sdk.asf_search.get_metadata is depreciated. We recommend using asf_search.granule_search '
+                  'instead. See: https://github.com/asfadmin/Discovery-asf_search#usage', FutureWarning)
+
     if isinstance(granules, str):
         granule_list = granules
     else:
         granule_list = ','.join(granules)
 
     params = {
-        'output': 'jsonlite',
+        'output': 'geojson',  # preferred by DISCOVERY-asf_search
         'granule_list': granule_list,
     }
     response = requests.post(_SEARCH_API, params=params)
     _raise_for_search_status(response)
 
-    metadata = [result for result in response.json()['results']
-                if not result['productType'].startswith('METADATA_')]
-
+    scenes = [result for result in response.json()['features']
+              if not result['properties']['processingLevel'].startswith('METADATA_')]
     if isinstance(granules, str):
-        return metadata[0]
+        return scenes[0]
 
-    return metadata
+    return {'features': scenes, 'type': 'FeatureCollection'}
 
 
-def get_nearest_neighbors(granule: str, max_neighbors: int = 2,) -> List[dict]:
+def get_nearest_neighbors(granule: str, max_neighbors: int = 2,) -> dict:
     """Get a Sentinel-1 granule's nearest neighbors from a temporal stack (backwards in time)
 
     Args:
         granule: reference granule
         max_neighbors: maximum number of neighbors to return
 
     Returns:
-        neighbors: a list of neighbors sorted by time
+        neighbors: GeoJSON FeatureCollection with a list of features for each neighbor, sorted by time
     """
     params = {
-        'output': 'json',  # jsonlite doesn't include centerLat/centerLon
+        'output': 'geojson',  # preferred by DISCOVERY-asf_search
         'platform': 'S1',
         'granule_list': granule,
     }
     response = requests.post(_SEARCH_API, params=params)
     _raise_for_search_status(response)
-    references = [r for r in response.json()[0] if not r['processingLevel'].startswith('METADATA_')]
+    references = [r for r in response.json()['features']
+                  if not r['properties']['processingLevel'].startswith('METADATA_')]
     if not references:
         raise ASFSearchError(f'Reference Sentinel-1 granule {granule} could not be found')
     reference = references[0]
+    reference_center_wkt = shape(reference['geometry']).centroid.wkt
+
+    from pprint import pprint as pp
+    pp(reference)
+    pp(reference_center_wkt)
```

Comment:
A number of `pp()` calls can be removed in this file

Reply:
:eyes: woops!